_sass/layout/_search.scss

```
@@ -0,0 +1,23 @@
+.search__wrapper {
+  margin-top: 20px;
+  margin-bottom: 50px;
+}
+
+.searchbar {
+  margin-bottom: 10px;
+
+  input {
+    height: 20px;
+  }
+}
```

Note here - `$spacing-base` should be used for _spacing_ within the app, and the `input` height doesn't quite fit that requirement. Instead, we should use `rem`s instead of pixels, since `rem`s are dynamic. Using pixels should be a last resort to get an exact measurement that you can't achieve with other methods.

```suggestion
.searchbar {
  margin-bottom: $spacing-base;

  input {
    height: 3rem;
  }
}
```